src/woo_publications/fixtures/informatie_category.json

```diff
@@ -0,0 +1,236 @@
+[
+{
+    "model": "metadata.informationcategory",
+    "pk": 1,
```

Comment:
the fixture should be dumped with natural keys so that primary keys don't end up in the fixture, that *will* cause problems with existing instances and when the fixture gains new records over time